python/paddle/fluid/dygraph/base.py

```diff
@@ -14,22 +14,20 @@
 from ..wrapped_decorator import signature_safe_contextmanager, wrap_decorator
 import contextlib
 import numpy as np
-import os
-
 from paddle.fluid import core
 from paddle.fluid import framework
 from .tracer import Tracer
 import logging
 
 __all__ = [
-    'enabled',
     'no_grad',
     'guard',
     'to_variable',
 ]
 
 
 def enabled():
```

Comment:
Add a TODO, we need to remove this since it always cause recurrent dependence

Reply:
ok